plugins/firehydrant/src/components/ServiceDetailsCard/ServiceDetailsCard.tsx

```diff
@@ -0,0 +1,285 @@
+import { useServiceDetails } from '../serviceDetails';
+import { useServiceAnalytics } from '../serviceAnalytics';
+import { InfoCard, Progress } from '@backstage/core-components';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
+
+const useStyles = makeStyles(theme => ({
+  infoCard: {
+    height: '342px',
+  },
+  button: {
+    color: '#3b2492',
+    display: 'grid',
+    gridGap: '4px',
+    textAlign: 'center',
+    justifyItems: 'center',
+    width: '102px',
+    backgroundColor: theme.palette.type === 'dark' ? '#f1edff' : '',
+    '&:hover, &:focus': {
+      backgroundColor: '#f1edff',
+      color: '#614ab6',
+    },
+    '&:active': {
+      color: '#3b2492',
+      backgroundColor: '#b2a6e3',
+      boxShadow:
+        'rgb(59, 36, 146) 0px 0px 0px 1px inset, rgb(141, 134, 188) 3px 3px 0px 0px inset;',
+    },
+    border: '1px solid #3b2492',
+    borderRadius: '5px',
+    padding: '10px',
+  },
+  buttonLink: {
+    backgroundColor: '#3b2492',
+    color: '#FFF',
+    textTransform: 'none',
+    '&:hover': {
+      backgroundColor: '#614ab6',
+    },
+  },
+  buttonContainer: {
+    display: 'grid',
+    gridGap: '24px',
+    gridAutoFlow: 'column',
+    gridAutoColumns: 'min-content',
+  },
+  icon: {
+    color: '#f1642d',
+  },
+  link: {
+    textDecoration: 'underline',
+    fontSize: '16px',
+    lineHeight: '27px',
+    color: '#3b2492',
+    '&:hover, &:focus': {
+      fontWeight: '500',
+    },
+  },
+  linksContainer: {
+    borderBottom: '1px solid #d5d5d5',
+    padding: '10px 0px 10px 20px',
+    backgroundColor: '#f1edff',
+    marginBottom: '20px',
+  },
+  table: {
+    width: '100%',
+  },
+  warning: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: '10px',
+    background: '#f1edff',
+    color: theme.palette.type === 'dark' ? '#3b2492' : '#fff',
+  },
+}));
+
+const ServiceWarning = ({ text }: { text: string }) => {
+  const classes = useStyles();
+  return (
+    <div className={classes.warning}>
+      <WarningIcon />
+      &nbsp;&nbsp;<span>{text}</span>
+    </div>
+  );
+};
+
+const ServiceAnalyticsView = ({
+  serviceId,
+  startDate,
+  endDate,
+}: {
+  serviceId: string;
+  startDate: moment.Moment;
+  endDate: moment.Moment;
+}) => {
+  const {
+    loading: analyticsLoading,
+    value: analyticsValue = {},
+    error: analyticsError,
+  } = useServiceAnalytics({
+    serviceId,
+    startDate: startDate.format('YYYY-MM-DD'),
+    endDate: endDate.format('YYYY-MM-DD'),
+  });
+
+  return (
+    <ServiceAnalytics
+      loading={analyticsLoading}
+      value={analyticsValue}
+      error={analyticsError}
+    />
+  );
+};
+
+export const ServiceDetailsCard = () => {
+  const { entity } = useEntity();
+  const classes = useStyles();
+  const [showServiceDetails, setShowServiceDetails] = useState(false);
+  const configApi = useApi(configApiRef);
+
+  const BASE_URL =
+    configApi.getOptionalString('firehydrant.baseUrl') ||
+    'https://app.firehydrant.io';
```

Comment:
Is there a self hosted Firehydrant offering?

Reply:
No there isn't, this was added for local development (left a comment above). Let me know if this should be removed